src/pages/EarnTri/index.tsx

```diff
@@ -59,10 +61,17 @@ export default function Earn({
   const [searchQuery, setSearchQuery] = useState<string>('')
   const [sortDescending, setSortDescending] = useState<boolean>(true)
 
+  const stableFarmsTabActive = farmsType === 'stable'
+  const tabFarmArrs = !stableFarmsTabActive
+    ? allFarmArrs.filter(farm => !farm.isStableSwap === true)
+    : allFarmArrs.filter(farm => farm.isStableSwap === true)
+
   const getSortedFarms = () => {
     switch (sortBy) {
       case SortingType.default:
-        return POOLS_ORDER.map(index => allFarmArrs[index])
+        return stableFarmsTabActive
+          ? farmArrs
+          : farmArrs.sort((a, b) => POOLS_ORDER.indexOf(a.ID) - POOLS_ORDER.indexOf(b.ID))
```

Comment:
Raw sort makes it hard for react
```suggestion
        return stableFarmsTabActive
          ? farmArrs
          : farmArrs.slice().sort((a, b) => POOLS_ORDER.indexOf(a.ID) - POOLS_ORDER.indexOf(b.ID))
```